news/2023-01-01-osumonthly-6.md

```diff
@@ -0,0 +1,331 @@
+
+Finally, we'd like to highlight [CrewK](https://osu.ppy.sh/users/11488604) as one of the best SV players the game has seen in the past couple of years. You can scroll over any SV map currently Loved or even Ranked, and I assure you you'll find them there. This year's highlight however is on [HowToPlayLN's polyriddim FC](https://osu.ppy.sh/beatmapsets/1182702#mania/2769068), specifically the play on the *brrrrrrrrrrrrrr vip* difficulty. To summarize it, it's timing hell. A memory gauntlet to behold, only worsened by its unforgiving patterning. It's hard to put into words, so please, feel free to check out CrewK's awesome and unique full clear on the map!
+
+You can view all of the plays listed in the playlist below:
+
+<iframe width="100%" height="315" src="https://www.youtube-nocookie.com/embed/videoseries?list=PLbY6YsbyBvHD3K_akt1j22zxeOixasi3f" frameborder="0" allowfullscreen></iframe>
+
+## Mapping recap
+
+All year long, the mappers and people involved in mapping on osu! have not stopped doing amazing things, and this month was no exception. So we will try to summarize all the things related to mapping that happened this month as best as possible in a few sentences.
+
+### Mapping News
+
+#### Relief of requirements to apply for BN
+
+A big change has taken place in the application requirements of Beatmaps Nominators. Indeed, the notion of ModScore has been totally removed (which wasa score based on the number of recent mods) from the required conditions evaluated during the application of future BNs. A way to encourage mappers and modders in the community to get involved in the ranking process by joining the BNG, now more accessible.
+
+#### SEV handbook added to the wiki
+
+The way in which the BNs were judged could seem incomprehensible sometimes for the general public until very recently, since the [SEV rating](/wiki/People/The_Team/Nomination_Assessment_Team/SEV_rating) has been added in the wiki. From the different punishable omissions, to the severity applied to each, everything is now detailed there (although in reality, SEV rating adapts to each case).
+
+### Mapping highlight
+
+Although, like most osu! players, you may be content with the beatmaps already installed on your computer, new extraordinary and trendy beatmaps are released every month and pass the ranked status. So we decided to add this new section to the monthly to keep you up to date with the most popular beatmaps among mappers, players and even music enthusiasts.
+
+By the way, even if we don't mention all the beatmaps that have been ranked this month, you should know that they are all great, and worth a visit. So don't hesitate to take a look at the beatmap section of the osu! website and check out the most recent beatmaps!
+
+#### Mighty Morphin Power Rangers by Sytho - Community's favorite
```

Comment:
`{artist} - {title} mapped by {mapper} - {category}` for these ty

Reply:
https://github.com/ppy/osu-wiki/pull/8570/commits/56a855c547e8d29a8d857d7c51a1de9310ff21a2